data/hai/hai reveal 4 middle.txt

```diff
@@ -1233,29 +1233,77 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	Having finished your main meal, and while waiting for some dessert that Teeneep has ordered, Sayari pulls out her datapad. "You know, while we're waiting. It's probably a good idea to check the damage. I don't know if your plan is actually going to have the desired effect, you know. I mean, seriously, Tee, what will you do if it doesn't? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
-			`	Teeneep just sips her drink and shakes her head. "Oh, Sayari, you really are getting too long in the tooth for domestic politics these days."`
-			`	A moment later Sayari squeaks and looks up in horror. "You triggered an election?! This wasn't just making it public, you had the petition for redress ready to go all along!"`
-			`	She scrolls down on her pad several times and begins to mutter, "Oh dear, oh no." She lapses into Hai for a moment before returning. "It hit the trending issues board, and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action and it's only been two hours... this is what you hoped would happen, isn't it?"`
-			`Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
-			label alarm
-			`	Sayari's eyes are alive with thinly-veiled panic as she slams her hand down on the table. "Captain <last>, you must leave immediately. Get on your ship and fly to <planet> right now! The winds of change are afoot and we will have an election. They'll hit the threshold in maybe only twenty minutes, and it'll take maybe thirty more for the bureaucracy to act on it. The instant the election is announced formally, all non-essential ships will be grounded, all communication in or out of Hai space will be cut off, and the border will be sealed to everyone - not in the polite way either. If even one ship tries to cross the border to investigate... there's no telling what will happen."`
+			`Upon your return, Teeneep sends an urgent demand for an audience with the Elders, along with a few paragraphs written in Hai. The meeting is called within an hour, and the room is entirely full. This may be the entire Elder Council; if so, they must consider this a grave matter. Teeneep and Sayari are waiting for you to follow them into the chambers.`
+			`	The Elder moderating the proceedings calls Teeneep to testify, using your language. It seems they consider your participation critical.`
+			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. They are ready to send a fleet to human space and extinguish the pirate threat. Will you do what is right for our people and negotiate this with the humans, or will you be replaced?"`
```

Comment:
I went to make three different suggestions but none of them were right.

This opening sentence has the following problems:

- It was less than 300 years (also we're trying not to be too specific about that because the date the player gets to this varies).
- If this issue is ongoing/cyclical then it's not the first time, and describing it as such weakens her position.
- We have not necessarily established why the council is inactive, or at least not presented the possible motivations (I don't think - feel free to correct me) which makes this accusation incredibly rude.
- Also, it's bad politic to directly insult the people you wish to galvanise to action when you've already indirectly insulted them by working behind their backs. This current presentation implies she *wants* them to refuse her...
- The Unfettered never stopped wanting to be defenders of all Hai, the only thing they've been convinced of is a willingness to properly fight the Hai over the moral issue of inaction.
- Also, most Unfettered alive were probably not in fact the actual defenders. They'd be a minority at this stage.
- Extinguishing the pirate threat entirely is not really possible without mass conducting atrocities, which the Elders know.
- What they've already been doing could well be described as "negotiating with humans", what's being asked is for them to demand action from specific humans (The Republic).

These Elders have been politicking for a long time. I wouldn't imagine that Teeneep would get to the end of this paragraph successfully without being interrupted a half dozen times and likely dismissed by half those assembled.